test/runtests.jl

```diff
@@ -114,6 +109,8 @@ end
 # Testsets.
 #
 
+tokentest
```

Comment:
Is this supposed to be a comment?

Reply:
Sorry that was left from testing, I'll remove it